packages/neo-one-node-http-rpc/src/rpcServer$.ts

```diff
@@ -105,12 +105,12 @@ export const rpcServer$ = ({
         router.get(readyMiddleware.name, readyMiddleware.path, readyMiddleware.middleware);
       }
 
-      if (rateLimitOptions !== undefined && rateLimitOptions.enabled) {
-        router.use(rateLimit(rateLimitOptions));
+      if (rateLimitOptions === undefined || rateLimitOptions.enabled) {
+        router.use(rateLimit({ rate: 6000 }));
```

Comment:
With much testing, this was found to be the sweet spot for tests to run successfully